octoprint_preheat/__init__.py

```diff
@@ -245,7 +245,16 @@ def on_api_command(self, command, data):
 				self._logger.info("Preheat error: " + str(error.message))
 				return str(error.message), 405
 
-				
+	def gcode_script_variables(self, comm, script_type, script_name, *args, **kwargs):
+		if not script_type == "gcode" or not script_name == "beforePrintStarted":
```

Comment:
Is there a reason for checking the script name? IMO accessing the variables could also be useful in the other scripts, for example to achieve preheating when a print is cancelled.